orchagent/orch.cpp

```diff
@@ -16,12 +13,7 @@
 
 using namespace swss;
 
-extern int gBatchSize;
-
-extern bool gSwssRecord;
-extern ofstream gRecordOfs;
-extern bool gLogRotate;
-extern string gRecordFile;
+int gBatchSize = 0;
```

Comment:
I see you move the definition of this global variable from main.cpp to orch.cpp. Is it really necessary?

Reply:
Yes, if not every daemon that inherits from Orch should define this. 

None of the cfgmgrs require the use of gBatchSize var, so it is defined as zero. Only orchagent needs to updated this and thus it is linked with "extern" and update in main.cpp